Data/ByteString/Short/Internal.hs

```diff
@@ -357,6 +362,12 @@ instance Data ShortByteString where
   gunfold _ _    = error "Data.ByteString.Short.ShortByteString.gunfold"
   dataTypeOf _   = mkNoRepType "Data.ByteString.Short.ShortByteString"
 
+packConstr :: Constr
+packConstr = mkConstr byteStringDataType "pack" [] Prefix
+
+byteStringDataType :: DataType
```

Comment:
`shortByteStringDataType` would be better. Is it used in `instance Data ShortByteString`?

Reply:
Changed, I also forgot to add the function to the `toConstr` instance in this file, that is now added